pg_search/src/writer/index.rs

```diff
@@ -121,7 +211,7 @@ impl Writer {
         Ok(())
     }
 
-    fn vacuum(&mut self, directory: WriterDirectory) -> Result<(), IndexError> {
+    pub fn vacuum(&mut self, directory: WriterDirectory) -> Result<(), IndexError> {
```

Comment:
surely vacuum support needs some kind of locking too since it also modifies the index.